PeriodogramAnalysis/PeriodogramAnalysis.py

```diff
@@ -5,35 +5,33 @@
 from scipy import signal as sig
 from scipy.integrate import simpson
 from dataclasses import dataclass, field
-from typing import Optional, Union
+from collections import defaultdict
 
 from miv.core.operator.operator import OperatorMixin
 from miv.core.operator.wrapper import cache_call
 from miv.typing import SignalType
-from PowerSpectrumAnalysis import SpectrumAnalysis, multitaper_psd
+from PowerSpectrumAnalysis import multitaper_psd
 
-@ dataclass()
+@dataclass
 class PeriodogramAnalysis(OperatorMixin):
     """
        A class to perform Periodogram Analysis using Welch's method and multitaper PSD.
```

Comment:
Can you put the reference for this analysis?

Reply:
The text is wrong. You didn't document the information on that page yourself.
Just say `the algorithm implementation is inspired from ...`.

Reply:
I see. 
Have changed.